src/common/time/src/timestamp.rs

```diff
@@ -789,4 +840,41 @@ mod tests {
             Timestamp::new(1, TimeUnit::Second).convert_to_ceil(TimeUnit::Millisecond)
         );
     }
+
+    #[test]
+    fn test_split_overflow() {
+        let _ = Timestamp::new(i64::MAX, TimeUnit::Second).split();
```

Comment:
Could we assert the result here? Is it necessary to ignore the result explicitly?
```suggestion
        Timestamp::new(i64::MAX, TimeUnit::Second).split();
```